packaging/docker/Dockerfile

```diff
@@ -43,6 +43,7 @@ RUN if [ "$(uname -m)" == "x86_64" ]; then \
         apk update; \
         apk upgrade --available --no-cache; \
         sed -in 's/^#//g' /etc/apk/repositories; \
+        apk fix py2-psycopg2; \
```

Comment:
Add a comment here please, referencing the issue number also that triggered this addition.
Just to be sure its not forgotten why its there